src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheck.java

```diff
@@ -113,6 +113,7 @@ public int[] getAcceptableTokens() {
             TokenTypes.STRING_LITERAL,
             TokenTypes.CHAR_LITERAL,
             TokenTypes.TEXT_BLOCK_CONTENT,
+            TokenTypes.STRING_TEMPLATE_CONTENT,
```

Comment:
please create separate issue on Check extension, we usually very transperent to users in release notes if some Check got new acceptable token to operate on.
So you will have 3 commits: minor, grammar, check update. All with different prefixes.

Reply:
Done